### DISCUSSION TITLE
I wanted to know update of issue #161 : Overflow of homepage

### DISCUSSION THREAD

**Discussion by ujjawaltyagii** [rupali-codes/LinksHub#254] [General]
## There is this issue that home page is scrollable which shouldn't be the case so is this issue idle ? or the assigned contributor has a plan because i don't see any update from him in his PR

https://user-images.githubusercontent.com/115401171/221358886-45d9eb58-1a3e-4b36-bf20-1cf37fecc23b.mp4

**Comment by Jay-Karia** [comment]
The issue #161 is fixed.
